app/celery/process_ses_receipts_tasks.py

```diff
@@ -31,6 +31,9 @@
 def process_ses_results(self, response):
     try:
         ses_message = json.loads(response["Message"])
+        current_app.logger.info(
+            f"Async response from SES when sending email (messageId: {ses_message['mail']['messageId']}): {ses_message}"
+        )
```

Comment:
Let's see in staging what kind of information is dumped there and make sure no sensitive data is leaked when hitting production.

Reply:
Ah good point.  I'll do some testing in staging and see what gets dumped out!